lib/resources/r4/slot_patch.yaml

```diff
@@ -0,0 +1,16 @@
+---
+name: Related Person
+field_name_base_url: https://hl7.org/fhir/r4/related-person-definitions.html#RelatedPerson
+operations:
+  - name: status
+    path: /status
+    operation: replace
+    type: code
+    url: https://hl7.org/fhir/R4/slot-definitions.html#Slot.status
+    description: Replace the slot status.
+    example: |
+      {
+        "status": "free"
+      }
```

Comment:
This should have op and path on it as well.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/608/commits/436435c75110e6b389fd06d812050eb025b4cfca#diff-8a92cfc782cbec26cd0d479d0184b0a249e166900d9f3bbef43ea04612760a9aR13